.gitignore

```diff
@@ -28,3 +28,6 @@ vendor/ruby
 
 # Generated js bundles
 /app/assets/javascripts/generated/*
+
+# Rubymine/IntelliJ
+.idea
```

Comment:
@dylangrafmyre If you work on a lot of open source, then you still need to exclude a lot of Mac and RubyMine specific files in your global gitignore. Adding these are harmless, but there are many projects you will check out (even git clone and open in RubyMine/Webstorm), and you don't want to be modifying other people's projects .gitignore all the time. I'm OK with this change here, however.